FsAutoComplete/CompilerServiceInterface.fs

```diff
@@ -0,0 +1,159 @@
+namespace FsAutoComplete
+
+open System
+open System.IO
+open Microsoft.FSharp.Compiler
+open Microsoft.FSharp.Compiler.SourceCodeServices
+
+type ParseAndCheckResults(parse: FSharpParseFileResults,
+                          check: FSharpCheckFileResults,
+                          version: int) =
+  let parseResults = parse
+  let checkResults = check
+  let _version = version
+
+  member x.TryGetMethodOverrides (lines: string[]) (line: int) (col: int) =
+    // Find the starting point, ideally right after the first '('
+    let lineCutoff = line - 3
```

Comment:
Yes, it is. @rojepp added this and I asked for the limit to be put on in case it ended up going all the way back to the start of the file. Perhaps 3 is a bit conservative.